host/CMakeLists.txt

```diff
@@ -178,24 +180,44 @@ target_include_directories(oehost PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 
 # TODO: Handle Trust Zone etc.
 if (USE_LIBSGX)
-  find_library(LIBSGX_COMMON NAMES sgx_enclave_common)
-  find_library(LIBSGX_QE NAMES sgx_dcap_ql)
-  find_library(LIBSGX_URTS NAMES sgx_urts)
+  if (UNIX)
 
-  if (NOT LIBSGX_COMMON OR NOT LIBSGX_QE OR NOT LIBSGX_URTS)
-    message(FATAL_ERROR "No SGX libraries found, aborting! Set -DUSE_LIBSGX=OFF to ignore.")
+    find_library(LIBSGX_COMMON NAMES sgx_enclave_common)
+    find_library(LIBSGX_QE NAMES sgx_dcap_ql)
+    if (NOT LIBSGX_COMMON OR NOT LIBSGX_QE)
+      message(FATAL_ERROR "No SGX libraries found, aborting! Set -DUSE_LIBSGX=OFF to ignore.")
+    endif ()
+
+    add_library(sgx_enclave_common SHARED IMPORTED)
+    set_target_properties(sgx_enclave_common PROPERTIES IMPORTED_LOCATION ${LIBSGX_COMMON})
+
+    add_library(sgx_dcap_ql SHARED IMPORTED)
+    set_target_properties(sgx_dcap_ql PROPERTIES IMPORTED_LOCATION ${LIBSGX_QE})
+
+  elseif (WIN32)
+
+    find_library(LIBSGX_COMMON NAMES sgx_enclave_common
+      PATHS ${CMAKE_SOURCE_DIR}/prereqs/nuget/EnclaveCommonAPI/lib/native/x64-Release)
```

Comment:
We shouldn't need to duplicate all this code, just add this `PATHS` conditionally under the above logic.

Reply:
Made some changes to reduce duplication to the extent possible. @andschwa - Please review.